### PR TITLE
Added better image scaling so that characters have consistent sizes

### PR DESCRIPTION
**Pull request by skyheat**
Characters within the game will now have more consistent sizes which helps with the visibility issues we were having.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "dev": "concurrently --kill-others-on-fail \"yarn client\" \"yarn server\""
   },
   "dependencies": {
+    "caniuse-lite": "^1.0.30001240",
```
Comment (Drewbi) [inline package.json:20]: Was this addition intentional?

File: client/src/Pages/Game/index.tsx [hunk 2]
```diff
@@ -85,11 +85,17 @@ const Game: React.FC = () => {
 
     setTimeout(() => {
       Konva.Image.fromURL(randomChar.image, function (image: any) {
+
+        // Scaling image based on original size. Currently uses a hacky way to do it but made the images more consistent in size
+        var imageHeight = image.attrs.image.height
+        var desiredHeight = 169
+        var scale = desiredHeight/imageHeight
```
Comment (Drewbi) [inline client/src/Pages/Game/index.tsx:92]: These can all be `const` as they are not being reassigned. For reassigned variables you can use `let`

**Review by Drewbi — APPROVED** [review]
Looks good, just some minor suggestions

File: package.json (hunk 1 above)
Comment (skyheat) [reply package.json:20]: Not too sure, as on initial run yarn dev threw some errors and told me to update caniuse-lite :/

Comment (Drewbi) [reply package.json:20]: Ah yep, all good

**Review by Drewbi — APPROVED** [review]
